.clj-kondo/babashka/sci/sci/core.clj

```diff
@@ -0,0 +1,9 @@
+(ns sci.core)
```

Comment:
Probably, you accidentally commited some clojure changes

Reply:
definitely did, though I'm not sure where they came from :/